lib/views/doujin/doujin_library/library_view.dart

```diff
@@ -0,0 +1,200 @@
+            id: 1234,
+            title: null,
+            tags: null),
+        Doujin(
+            dateAdded: 1023,
+            uploadDate: 1231,
+            cover: 'dasd',
+            scanlator: null,
+            rating: 3,
+            mediaId: '1234647',
+            id: 1234,
+            title: null,
+            tags: null),
+        Doujin(
+            dateAdded: 1023,
+            uploadDate: 1231,
+            cover: 'dasd',
+            scanlator: null,
+            rating: 3,
+            mediaId: '1234647',
+            id: 1234,
+            title: null,
+            tags: null),
+        Doujin(
+            dateAdded: 1023,
+            uploadDate: 1231,
+            cover: 'dasd',
+            scanlator: null,
+            rating: 3,
+            mediaId: '1234647',
+            id: 1234,
+            title: null,
+            tags: null),
+      ];
+    });
+
+    // TODO: for real use case
+    /*var doujinProvider = DoujinProvider();
+
+    doujinProvider.retrieveAll().then((value) {
+      setState(() {
+        doujins = value;
+        isLoading = false;
+      });
+    });*/
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    _retrieveDoujins();
+  }
+
+  @override
+  void deactivate() {
+    super.deactivate();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(widget.title),
+      ),
+      drawer: Drawer(
+        child: ListView(
+          padding: EdgeInsets.zero,
+          children: const <Widget>[
+            DrawerHeader(
+              decoration: BoxDecoration(color: Colors.deepPurple),
+              child: Text(
+                'Menu',
+                style: TextStyle(
+                  color: Colors.white,
+                  fontSize: 24,
+                ),
+              ),
+            ),
+            ListTile(
+              leading: Icon(Icons.message),
+              title: Text('Messages'),
+            ),
+            ListTile(
+              leading: Icon(Icons.account_circle),
+              title: Text('Profile'),
+            ),
+            ListTile(
+              leading: Icon(Icons.settings),
+              title: Text('Settings'),
+            ),
+          ],
+        ),
+      ),
+      body: GridView.builder(
+        gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+          crossAxisCount: 2,
+          mainAxisSpacing: 1.0,
+          crossAxisSpacing: 2.0,
+          childAspectRatio: MediaQuery.of(context).size.width /
+              (MediaQuery.of(context).size.height / 1.3),
+        ),
+        itemCount: doujins?.length ?? 0,
+        itemBuilder: (context, index) {
+          final item = doujins?[index];
+
+          return GridTile(
+              child: DoujinCards(
+                  cover: '123123',
+                  mediaId: '123122',
+                  title: 'hasbdyb asd vb',
+                  rating: 4,
+                  id: 123123));
```

Comment:
add a comma to this double brackets too, but in this case add one more comma to the end of the last parameter, i.e `id: 123123,` so that the brackets will go to the next line.